sdk/identity/identity/src/client/identityClient.ts

```diff
@@ -308,4 +308,10 @@ export interface TokenCredentialOptions extends PipelineOptions {
    * The default is "https://login.microsoftonline.com".
    */
   authorityHost?: string;
+  /**
+   * Optionally, a single correlationId can be configured as the default `correlationId`
+   * for all of the outgoing requests. This value can be replaced by specifying a custom
+   * `correlationId` on the options sent to the `getToken()` method (or the `authenticate()` method).
```

Comment:
I don't see correlationId in the [GetTokenOptions](https://docs.microsoft.com/en-us/javascript/api/@azure/core-auth/gettokenoptions?view=azure-node-latest) interface. Where do we define that getToken() can take this? Or is this applicable to only certain credentials?

Reply:
Oof that's a great point! I completely missed this.

Reply:
PR: https://github.com/Azure/azure-sdk-for-js/pull/14877

Reply:
Merged that PR since Karishma and Jeff approved.